docs/getting-started.md

```diff
@@ -232,7 +224,682 @@ curl localhost:10101/index/repository/query \
 Please note that while user ID 99999 may not be sequential with the other column IDs, it is still a relatively low number. 
 Don't try to use arbitrary 64-bit integers as column or row IDs in Pilosa - this will lead to problems such as poor performance and out of memory errors.
 
+#### Using Go
+
+Pilosa requires Go 1.12 or higher. It is also recommended that you have a code editor downloaded.
+
+##### Create the Environment
+
+In order to communicate with Pilosa through your Go code, you must have a "translator," which is go-pilosa. To install go-pilosa, open a terminal (one other than the one running Pilosa) and download the library in your `GOPATH` using:
+``` 
+go get github.com/pilosa/go-pilosa
+```
+
+To contain the Getting Started project in one place, we will create a new folder as follows:
+```
+mkdir GettingStarted && cd GettingStarted
+```
+
+In this folder, we will download two CSV files to provide data to our fields later on. Download the `stargazer.csv` and `language.csv` files here:
+```
+curl -O https://raw.githubusercontent.com/pilosa/getting-started/master/stargazer.csv
+curl -O https://raw.githubusercontent.com/pilosa/getting-started/master/language.csv
+```
+
+We will also create a file called `StarTrace.go` as follows:
+```
+touch StarTrace.go
+```
+This file will be used in the following sections.
+
+##### Create the Schema
+
+Before we can import data or run queries, we need to create our schema. Go-pilosa is implemented by importing `github.com/pilosa/go-pilosa` and its ability to read csv files is implemented by importing 'github.com/pilosa/go-pilosa/csv`. The first steps to creating the schema are creating a client which will communicate our schema to Pilosa, creating a schema which will contain our indexes and fields, and syncing with Pilosa. This is all done in the `StarTrace.go` file:
```

Comment:
I think those three steps are *all* of the steps in creating the schema - I'd phrase this as something like "Create the schema by first creating a client, defining the schema locally, then syncing with Pilosa".